docs/BestPractices.md

```diff
@@ -23,6 +23,13 @@ FROM node:4.1.2-onbuild
 RUN groupadd -r node && useradd -r -g node node
 ```
 
+When using the `alpine` variant, add the user like so:
+
+```Dockerfile
+FROM node:7.3.0-alpine
+RUN addgroup -S node && adduser -S -g node node 
+```
```

Comment:
See https://github.com/nodejs/docker-node/issues/263, and try adding `--pull` to your `docker build` invocation.